projects/aws/lib/aws-stack.ts

```diff
@@ -339,6 +352,12 @@ export class EditionsStack extends cdk.Stack {
             },
         )
 
+        archiveS3EventListener.addToRolePolicy(
+            new iam.PolicyStatement({
+                actions: ['sts:AssumeRole'],
```

Comment:
What's this for?

Reply:
For CMS fronts s3 Access so that s3 client can assume role for CMS fronts

Reply:
I didn't think the trigger needs CMS access?

Reply:
it needs now, because we get the metadata from json file content, not from the path